docs/en/operations/settings/settings.md

```diff
@@ -404,6 +404,32 @@ Possible values:
 
 Default value: 0.
 
+## any_join_distinct_right_table_keys {#any_join_distinct_right_table_keys}
+
+Enables or disables specific ClickHouse server behavior in `ANY INNER|LEFT|RIGHT| JOIN` operations.
```

Comment:
ANY LEFT has the same behaviour regardless of `any_join_distinct_right_table_keys`

Reply:
That means setting in `ANY LEFT` has no effect? It can be added in a separate comment.

Reply:
The case is either not list `INNER|LEFT|RIGHT` here either list without `LEFT`.